### PR TITLE
WIP: Campaign Pixel template tag

### PR DESCRIPTION
**Pull request by MichaelButkovic**

### CODE REVIEW

**Comment by MichaelButkovic** [comment]
this isn't finished, but i'm opening this up now to get feedback @spra85 @benghaziboy @mparent61

File: bulbs/campaigns/templatetags/campaign_tags.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+register = template.Library()
+
+
+@register.inclusion_tag('campaigns/_campaign_pixels.html')
+def campaign_pixels(*args, **kwargs):
+    campaign_id = kwargs['campaign']
+    property_id = kwargs['property']
+
+    params = {'campaign': campaign_id, 'property': property_id}
+    response = requests.get('http://www.tunic.theonion.com/api/v1/pixels', params=params)
```
Comment (mparent61) [inline bulbs/campaigns/templatetags/campaign_tags.py]: Grab URL from settings

File: bulbs/campaigns/templatetags/campaign_tags.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    valid_pixels = []
+
+    for pixel in response.json()['results']:
+        now = datetime.now(timezone('UTC'))
+
+        begun = parse_datetime(pixel['campaign_start_date']) < now
```
Comment (mparent61) [inline bulbs/campaigns/templatetags/campaign_tags.py]: Use <= for begun, generally we've been following convention:

start <= now < end

File: bulbs/campaigns/templatetags/campaign_tags.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    campaign_id = kwargs['campaign']
+    property_id = kwargs['property']
+
+    params = {'campaign': campaign_id, 'property': property_id}
+    response = requests.get('http://www.tunic.theonion.com/api/v1/pixels', params=params)
+
+    valid_pixels = []
+
+    for pixel in response.json()['results']:
+        now = datetime.now(timezone('UTC'))
```
Comment (mparent61) [inline bulbs/campaigns/templatetags/campaign_tags.py]: datetime.utcnow() is simpler (unless this does something extra?)

**Pull request closed without merging**